includes/class-wc-stripe-helper.php

```
@@ -1216,6 +1216,28 @@ public static function get_adaptive_pricing_account_unavailable_reason(): ?strin
+	public static function get_adaptive_pricing_unavailable_reason(): ?string {
+		if ( ! self::is_checkout_sessions_available() ) {
+			$is_automatic_capture_enabled = ( self::get_stripe_settings()['capture'] ?? 'yes' ) === 'yes';
+			if ( ! $is_automatic_capture_enabled ) {
+				return 'manual-capture';
+			}
+
+			return 'disabled';
+		}
+
+		return self::get_adaptive_pricing_account_unavailable_reason();
+	}
```

PHPMD (2.15.0)

[error] The method get_adaptive_pricing_unavailable_reason is not named in camelCase. (undefined)

(CamelCaseMethodName)